enterprise/moxiemanager.md

```diff
@@ -7,15 +7,13 @@ keywords: moxiemanager .net php relative_urls
 
 ## File and Image Management Using Moxie Manager
 
-{% include pardot-px-moxiemanager.html %}
+MoxieManager is a premium {{site.productname}} plugin and server system that enables users to insert files located externally to the editor (e.g. on their client desktop) into their document.
 
-MoxieManager is a premium TinyMCE plugin and server system that enables users to insert files located externally to the editor (e.g. on their client desktop) into their document.
+MoxieManager is an application separate from {{site.productname}} that includes .NET or PHP server components. Being built by the team behind {{site.productname}} it has tight integration with the editor via a toolbar control. It is designed to work with {{site.productname}} configuration settings such as `relative_urls`.
 
-MoxieManager is an application separate from TinyMCE that includes .NET or PHP server components. Being built by the team behind TinyMCE it has tight integration with the editor via a toolbar control. It is designed to work with TinyMCE configuration settings such as `relative_urls`.
+MoxieManager is a paid addition to {{site.productname}}. It is included in [{{site.cloudname}} subscriptions]({{site.pricingpage}}).
```

Comment:
This looks like the wrong variable here? As this was `TinyMCE subscriptions` previously, now it's `Tiny Cloud subscriptions` which I'm pretty sure MoxieManager doesn't come with cloud.

Reply:
fixed